ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDEvidenceType.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.cmc.ccd.domain;
+
+public enum CCDEvidenceType {
+    CONTRACTS_AND_AGREEMENTS("Contracts and agreements"),
+    EXPERT_WITNESS("Expert witness"),
+    CORRESPONDENCE("Letters, emails and other correspondence"),
+    PHOTO("Photo evidence"),
+    RECEIPTS("Receipts"),
+    STATEMENT_OF_ACCOUNT("Statements of account"),
+    OTHER("'Other'");
```

Comment:
are you sure this is "'Other'" not "Other"?

Reply:
fixed